doc/content/guides/connecting-gateways/ciscoixm/_index.md

```diff
@@ -0,0 +1,348 @@
++ `copy running-config startup-config` to save the configuration.
+
+#### Verification
+
+Before we install the packet forwarder, let's run verification to ensure that the gateway is ready.
+
++ Type `show radio` to verify that the **radio is enabled**. The result should indicate **radio status: on**.
++ Type `show inventory` to verify that the **FPGAStatus is Ready**.
++ Type `show gps status` to verify that the **GPS is correctly connected**. You can get additional GPS metadata by typing `show gps info`.
++ Verify that the **network connection is working**. You can test this by pinging common ping servers with `ping ip <IP>`, if your local network does not block ping commands. For example, you can ping Google's servers with `ping ip 8.8.8.8`.
+
+If some of those checks fail, go back to the appropriate section earlier in order to fix it.
+
+Then save the configuration by executing
+
+```
+Gateway# copy running-config startup-config
+```
+
+### Packet Forwarder Configuration
+
+> ⚠️ Keep in mind that the pre-installed packet forwarder is not supported by Cisco for production purposes.
+
+To run the packet forwarder, we'll make use of the **container** that is running on the gateway at all times.
+
+```
+Gateway# request shell container-console
+```
+You will be requested to enter the System Password. By default this is **admin**.
+
+Create the directory to store the Packet Forwarder configuration:
+
+```bash
+bash-3.2# mkdir /etc/pktfwd
+```
+Copy the packet forwarder to **/etc/pktfwd**:
+
+```bash
+bash-3.2# cp /tools/pkt_forwarder /etc/pktfwd/pkt_forwarder
+```
+
+Cisco provides a list of configuration templates. To list the available templates:
+
+```bash
+bash-3.2# ls -l /tools/templates
+  total 136
+  -rwxr-xr-x    1 65534    65534        11323 Oct  8 13:30 config_loc_dual_antenna_8ch_full_diversity_EU868.json
+  -rwxr-xr-x    1 65534    65534        11248 Oct  8 13:30 config_loc_dual_antenna_8ch_full_diversity_JP920.json
+  -rwxr-xr-x    1 65534    65534        11323 Oct  8 13:30 config_loc_dual_antenna_8ch_partial_diversity_EU868.json
+  -rwxr-xr-x    1 65534    65534         7993 Oct  8 13:30 config_loc_single_antenna_16ch_EU868.json
+  -rwxr-xr-x    1 65534    65534         7080 Oct  8 13:30 config_loc_single_antenna_16ch_US915.json
+  -rwxr-xr-x    1 65534    65534        13519 Oct  8 13:30 config_loc_single_antenna_64ch_US915.json
+  -rwxr-xr-x    1 65534    65534        13635 Oct  8 13:30 config_loc_single_antenna_full_duplex_64ch_US915.json
+  -rwxr-xr-x    1 65534    65534        17478 Oct  8 13:30 config_test_dual_antenna_56ch_partial_diversity_EU868.json
+  -rwxr-xr-x    1 65534    65534        14148 Oct  8 13:30 config_test_single_antenna_64ch_64x1_EU868.json
+  -rwxr-xr-x    1 65534    65534        14148 Oct  8 13:30 config_test_single_antenna_64ch_8x8_EU868.json
+```
+
+Copy the configuration template **config_loc_single_antenna_16ch_EU868.json** as **config.json** to **/etc/pktfwd** 
```

Comment:
This is wrong; 1) we never default to EU868 in our documentation because we do not favor the region where we happen to be from and 2) we have the GCS to generate JSON files for us, based on the actual frequency plan that is configured for the gateway.

Reply:
Ok, I will not default to `EU868` but should we keep it the old fashioned way while we work on the GCS generated JSON?